lib/widgets/app_launcher/app_laucher_tile.dart

```diff
@@ -23,7 +23,7 @@ import 'package:pangolin/utils/providers/search_provider.dart';
 
 class AppLauncherTile extends StatefulWidget {
   final String packageName;
-  const AppLauncherTile(this.packageName);
+  const AppLauncherTile(this.packageName, {Key? key}) : super(key: key);
```

Comment:
here you should have a required this.packageName because the field doesn't allow nullable types